codespell_lib/data/dictionary.txt

```diff
@@ -28809,6 +28814,7 @@ vissible->visible
 visted->visited
 visting->visiting
 vistors->visitors
+visualyse->visualize
```

Comment:
Can you suggest visualise first.

Can you also add an equivalent typo for visualiser/visualizer.

Reply:
Thanks, can you add the `visualyser` equivalent please.